src/Stock.js

```diff
@@ -13,6 +13,13 @@ class Stock extends React.Component {
             stockTimeValues: [],
             stockPriceValues: [],
             stockVolumeValues: [],
+            companyName: [],
+            company_52WeekHigh: [],
+            company_52WeekLow: [],
```

Comment:
Any reason these two have underscores while the others are just camel case? could probably just do company52WeekHigh etc. for consistency (and remember to change the other similarly named variables)

Reply:
Underscores have been removed!